modules/effects/arrow-effects-kotlinx-coroutines/src/main/kotlin/arrow/effects/DeferredK.kt

```diff
@@ -496,40 +496,48 @@ sealed class DeferredK<A>(
       CompletableDeferred<A>().apply { completeExceptionally(t) }.k()
 
     /**
-     * Starts a coroutine that'll run [Proc].
+     * Starts a coroutine that'll run [DeferredKProc].
      *
-     * Matching the behavior of [async],
+     * Matching the behavior of [asyncK],
      * its [CoroutineContext] is set to [DefaultDispatcher]
      * and its [CoroutineStart] is [CoroutineStart.LAZY].
      *
      * {: data-executable='true'}
      *
      * ```kotlin:ank
      * import arrow.core.Either
-     * import arrow.core.left
+     * import arrow.core.right
      * import arrow.effects.DeferredK
+     * import arrow.effects.DeferredKConnection
      * import arrow.effects.unsafeAttemptSync
      *
+     * class Resource {
+     *   fun asyncRead(f: (String) -> Unit): Unit = f("Some value of a resource")
+     *   fun close(): Unit = Unit
+     * }
+     *
      * fun main(args: Array<String>) {
      *   //sampleStart
-     *   val result = DeferredK.async { cb: (Either<Throwable, Int>) -> Unit ->
-     *     cb(
-     *       Exception("BOOM").left()
-     *     )
+     *   val result = DeferredK.async { conn: DeferredKConnection, cb: (Either<Throwable, String>) -> Unit ->
+     *     val resource = Resource()
+     *     conn.push(DeferredK { resource.close() })
+     *     resource.asyncRead { value -> cb(value.right()) }
```

Comment:
This is a probable cause for the explosion in CI. Ank can't handle asynchrony of any kind.